src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -150,7 +150,7 @@ private void initializeProducer() throws Exception {
   public void start() {
     if (_running.compareAndSet(false, true)) {
       initializeStateForPartitions();
-      _handleNewPartitionsExecutor.scheduleWithFixedDelay(new NewPartitionHandler(), 40_000, 40_000, TimeUnit.MILLISECONDS);
+      _handleNewPartitionsExecutor.scheduleWithFixedDelay(new NewPartitionHandler(), 1000, 30000, TimeUnit.MILLISECONDS);
```

Comment:
No.. Topic creation is different from partition reassignment. Multiple topics can be created concurrently. All we need to do to create a topic is to create znode `/brokers/topics/[topic]`. Since different KafkaMonitor should uses different topic, these znodes don't override each other.